### DISCUSSION TITLE
Map enhancement: Color systems by owner

### DISCUSSION THREAD

**Discussion by esksp** [endless-sky/endless-sky#10776] [Ideas]
Right now when you go to the map view ("M"), systems are colored by the number of planets you have visited. While this can be a useful information at times, quite often you're just looking to find the nearest civilization/species planet, and this is tedious and involves a lot of clicking.

~~It would be nice if **by clicking on the legend box** (top right under the system map), you could switch coloring between:~~
- ~~Current coloring (planets visited)~~
- ~~Owner (if any, gray for uninhabited), like "Republic", "Syndicate", "Hai", "Remnant", and so on~~
- ~~A simple friendly/hostile coloring (red= hostile, green= friendly, yellow=restricted, gray=neutral)~~

~~Just an idea.~~

**Edited**: That feature exists, but it is well hidden. Could you make it easier to find? Pretty pretty please?... :-D

**Comment by Saugia** [comment]
![image](https://github.com/user-attachments/assets/677b4e65-255e-44ab-a29e-ff25f4560c9a)

This already exists in-game; if you click on each part of the bottom-left panel, it switches what colors on the map represent things. Clicking the system name shows hostility, clicking the government shows government colors, and clicking on any of the trade options shows the colors there. The color-code panel you're talking about also changes based on which option you clicked on.

**Reply by esksp** [reply]
> Clicking the system name shows hostility, clicking the government shows government colors

OMG!...
That is almost impossible to guess!
I did of course try the obvious things, like clicking on the right panel (nothing), and on the left bottom panel (shows trading prices for selected ware). I never ever thought about clicking on the little _name_ tag panel...

Okay, lets change my suggestion to "make this a little easier to find, please"... 🙄

**Reply by warp-core** [reply]
I believe the help message includes reference to this functionality, which you should be able to bring up with the F1 key (or whatever you have "Show help" mapped to).

**Reply by esksp** [reply]
> I believe the help message includes reference to this functionality

OK, except there is no hint of there being a help menu, and F1 doesn't work in the main screen (planet view), so it isn't really obvious to find.  :-(

Is it so difficult to have that same thing (also?) happen by clicking on the map legend panel (the one which explains the colors)? It's the place people will naturally try to click on when hoping to get another kind of information inside.

BTW while we're at it, the map's civilization display has way too many, only very slightly different colors. I'm not color blind, but I have a hard time to keep some apart. I don't know what could be done here, but it's clear that while the first civilizations are easy to keep apart, at some point you guys ran out of colors...  :-D
Besides the map legend doesn't show all the civilizations on screen, it only shows a digest, depending on where the map is centered on. In most cases the map shows more civilizations than the legend panel can contain: A scrollbar would be nice I think.

**Reply by ziproot** [reply]
#10786

**Reply by Zireael07** [reply]
Re: the map civ colors, I believe there's an open issue for that already

**Reply by esksp** [reply]
> #10786

"While the game shows help messages, they are shown once": Not exactly the same issue, but probably aggravating the problem.

**My issue is about not being able to guess some "less intuitive" aspects of the game.**
This applies to new players, since experienced players usually already know their way around the game (unless they have forgotten...).

**Reply by esksp** [reply]
> Re: the map civ colors, I believe there's an open issue for that already

I am pretty sure I'm not the first to notice that...
Still, both aspects of this needs to be addressed somehow: The "not all civilizations are listed", and the "confusingly similar colors". For the first aspect, a scrollbar would help, for the second it's an old and well-known problem with maps, IIRC there are _ad hoc_ [solutions](https://en.wikipedia.org/wiki/Map_color_problem).
